set.mm

```diff
@@ -195655,6 +195673,30 @@ being prime ( ` Prime = { p e. NN | ... ` ), but even if ` p e. NN0 ` was
     cfv syl2anr wi prmfac1 3exp impcom con3d sylbid ) ACDZBEDZFZBAGHZABIHZJZABK
     SLHZJUHBMDAMDUJULNUGBOUGAAPQBARTUIUMUKUHUGUMUKUAUHUGUMUKABUBUCUDUEUF $.
 
+  ${
+    $( Condition for a prime number to divide a binomial coefficient.
+       (Contributed by Thierry Arnoux, 17-Sep-2023.) $)
+    prmdvdsbc $p |- ( ( P e. Prime /\ N e. ( 1 ... ( P - 1 ) ) ) -> P || ( P _C
+       N ) ) $=
```

Comment:
Usually I try to break the expression line along the most important symbol (here, `->`), sorry I did not do it here.
It would be easier to read with e.g.
```
prmdvdsbc $p |- ( ( P e. Prime /\ N e. ( 1 ... ( P - 1 ) ) )
              -> P || ( P _C N ) ) $=
```